### PR TITLE
Update icon/image documentation to be more clear

### PR DESCRIPTION
**Pull request by marksdk**

### CODE REVIEW

**Comment by marksdk** [comment]
@simonlaustsen We've had a bit of feedback regarding the use of "icon" vs "image" so I've changed it to only use "icon" everywhere.

File: collections/_cms/index.md [hunk 1]
```diff
@@ -266,6 +266,7 @@ Preview is especially helpful when using rotation or larger-than-monitor exports
 
 * Edit the Venue name in the available languages
 * Choose a default floor
+* Add an image of the Venue. Upload in a 600w x 300h size to make it look its best in our Standard Apps.
```
Comment (simonlaustsen) [inline collections/_cms/index.md:269]: Where in the CMS is it possible to add a venue image?

**Review by simonlaustsen — APPROVED** [review]
👍

File: collections/_cms/index.md (hunk 1 above)
Comment (marksdk) [reply collections/_cms/index.md:269]: Good catch!

**Review by simonlaustsen — APPROVED** [review]
👍

File: collections/_cms/index.md [hunk 2]
```diff
@@ -267,6 +267,8 @@ Preview is especially helpful when using rotation or larger-than-monitor exports
 * Edit the Venue name in the available languages
 * Choose a default floor
 
+_A note about Venue images:_ You can get MapsPeople to add an image of the Venue. It should be in a 600w x 300h size to make it look its best in our Standard Apps.
```
Comment (klm-mapspeople) [inline collections/_cms/index.md:270]:
```suggestion
_A note about Venue images:_ You can request MapsPeople to add an image of the Venue. It should be in a 600w x 300h size to make it look its best in our Standard Apps.
```

File: collections/_cms/index.md [hunk 3]
```diff
@@ -455,17 +457,17 @@ If you use Excel you can find a video on how to open a CSV file here: [https://w
...
+In the Icon Manager, you can choose between a predefined set of MapsIndoors icons, and a list of your uploaded icons. We support uploads of files in the PNG and SVG formats. Click on an icon in either place, and set it to close the icon manager and preview the icon on the map. Don't worry, it won't be saved until you click "Save" in the details editor.
 
-Remember to keep your icons at a reasonable size. We'll let you know if it's larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the image size of 900x600px or 150kb. The images should also be optimized and compressed to be as small as possible.
+Remember to keep your icons at a reasonable size. We'll let you know if it's larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```
Comment (klm-mapspeople) [inline collections/_cms/index.md:462]:
```suggestion
Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```

File: collections/_cms/index.md [hunk 4]
```diff
@@ -459,7 +459,7 @@ Anywhere in the CMS you can change icons for one or more markers on the map, you
 
 In the Icon Manager, you can choose between a predefined set of MapsIndoors icons, and a list of your uploaded icons. We support uploads of files in the PNG and SVG formats. Click on an icon in either place, and set it to close the icon manager and preview the icon on the map. Don't worry, it won't be saved until you click "Save" in the details editor.
 
-Remember to keep your icons at a reasonable size. We'll let you know if it's larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
+Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```
Comment (marksdk) [inline collections/_cms/index.md:462]:
```suggestion
Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon is larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```

**Pull request merged**